src/Altinn.Profile.Integrations/Migration/v0.04/03-setup-table.sql

```diff
@@ -0,0 +1,39 @@
+-- Scrips generated by "dotnet ef migrations script --idempotent --startup-project ../Altinn.Profile"
+
+-- Create table Organizations
+CREATE TABLE organization_notification_address.organizations (
+    registry_organization_id integer GENERATED BY DEFAULT AS IDENTITY,
+    registry_organization_number character varying(9) NOT NULL,
+    CONSTRAINT organization_id_pkey PRIMARY KEY (registry_organization_id)
+);
+
+-- Create table RegistrySyncMetadata
+CREATE TABLE organization_notification_address.registry_sync_metadata (
+    last_changed_id character varying(32) NOT NULL,
+    last_changed_date_time timestamp with time zone NOT NULL,
+    CONSTRAINT registry_sync_metadata_pkey PRIMARY KEY (last_changed_id)
+);
+
+-- Create table NotificationsAddresses
+CREATE TABLE organization_notification_address.notifications_address (
+    contact_info_id integer GENERATED BY DEFAULT AS IDENTITY,
+    registry_organization_id integer NOT NULL,
+    registry_id character varying(32) NOT NULL,
+    address_type integer NOT NULL,
+    domain character varying(200) NOT NULL,
+    address character varying(200) NOT NULL,
+    full_address character varying(200) NOT NULL,
+    created_date_time timestamp with time zone NOT NULL DEFAULT (now() at time zone 'utc'),
+    registry_updated_date_time timestamp with time zone,
+    update_source integer NOT NULL,
+    has_registry_accepted boolean,
+    is_soft_deleted boolean,
+    notification_name character varying(200),
+    CONSTRAINT contact_info_pkey PRIMARY KEY (contact_info_id),
+    CONSTRAINT registry_organization_id_akey UNIQUE (registry_organization_id),
+    CONSTRAINT "FK_notifications_address_organizations_OrganizationRegistryOrg~" FOREIGN KEY (registry_organization_id) REFERENCES organization_notification_address.organizations (registry_organization_id)
```

Comment:
Is that '~' added on purpose?

Reply:
Try to add an explicit name for the constraint defined in ProfiledbContext. 
See how it is was done for "krr":
`.HasConstraintName("fk_mailbox_supplier");`

Reply:
I'm finding the code very messy with all the annotations. Did you consider using a convention instead for KRR for snake case? 
such as https://github.com/efcore/EFCore.NamingConventions
CC: @Ahmed-Ghanam

Reply:
I'm thinking the tilde was EFs way of marking that the name is shortened. 

I don't like the KRR solution much as it is very manual instead of letting EF control the foreign keys.